internal/outofband/action_handlers.go

```diff
@@ -595,6 +595,7 @@ func (h *actionHandler) pollFirmwareTaskStatus(a sw.StateSwitch, c sw.Transition
 			action.Firmware.Component,
 			action.BMCTaskID,
 			action.Firmware.Version,
+			true, // need to be false for AsRockRack E3C256D4ID-NL
 		)
```

Comment:
I may be missing some context here, but how does this get set to false when flasher is handling a firmware install on a AsRockRack `E3C256D4ID-NL`?

Reply:
It doesn't for the moment.
Joel has some sort of special step for making this happen so I provide a way to set the boolean, defaulting to true, and then additional work will provide the rest.

Reply:
bmclib will indicate that the caller should not attempt to open a BMC session, Im still working on the implementation